### PR TITLE
add homepage style

### PR DESCRIPTION
**Pull request by pierrelg**

### CODE REVIEW

File: app/assets/javascripts/pages.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (mouligno) [inline app/assets/javascripts/pages.coffee]: Remove this file, coming from a scaffold. You don't need it.

File: app/helpers/pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module PagesHelper
```
Comment (mouligno) [inline app/helpers/pages_helper.rb]: Remove this file, coming from scaffold !

**Comment by mouligno** [comment]
Review done ! 

MRW i haven't seen any of the Star War movies and every one keeps asking me what i think about the new trailer

**Comment by pierrelg** [comment]
@mouligno remove bad files

**Comment by mouligno** [comment]
You can merge

**Pull request merged**